testsuite/src/it/java/org/graalvm/tests/integration/utils/versions/QuarkusVersion.java

```diff
@@ -34,11 +41,13 @@ public QuarkusVersion(String version) {
             this.major = Integer.parseInt(version.split("-")[0]);
             this.minor = 0;
             this.patch = 0;
+            this.gitSHA = getProperty("QUARKUS_VERSION_GITSHA", "");
```

Comment:
Where is this being set though? Does it happen in some script outside of this repo?

Reply:
It's the same as with all other properties, including `QUARKUS_VERSION` or e.g. `FAIL_ON_PERF_REGRESSION`, i.e. outside the repo, in the environment that runs the TS.

`QUARKUS_VERSION_GITSHA` specifically is currently used in a one test job only: [mandrel-linux-integration-tests-perf-comparator](https://ci.modcluster.io/view/Mandrel/job/mandrel-linux-integration-tests-perf-comparator/), [source](https://github.com/graalvm/mandrel-packaging/blob/master/jenkins/jobs/tests/Constants.groovy#L112), [job definition](https://github.com/graalvm/mandrel-packaging/blob/master/jenkins/jobs/tests/mandrel_linux_integration_tests_perf_comparator.groovy).